addons/hr/wizard/mail_activity_schedule.py

```diff
@@ -56,3 +78,10 @@ def _compute_department_id(self):
                 wizard.department_id = False if len(all_departments) > 1 else all_departments
             else:
                 wizard.department_id = False
+
+    def _get_activity_summary(self, record, template):
```

Comment:
This is trying to hack the system for 0 real use case. Please remove this override, it just add noise on the model.